node-fs-writefile/cp.js

```diff
@@ -0,0 +1,22 @@
+import { writeFile } from 'node:fs';
+import { readFile } from 'node:fs/promises';
+
+try {
+  const data = await readFile(String(process.argv[2]), 'utf8');
+  console.log(data);
+  writeFile('copy-note.txt', data, (err) => {
```

Comment:
Oh, it looks like you are using the non-promise version of writeFile! Sorry you got caught in that trap. You'll need to modify it to use the promise version, with async/await.

Looks like you did the same thing in the other two files. The code is all correct, other than it uses callbacks, so it shouldn't be hard to fix.

Reply:
@RobertGardner quick question about this. You may have covered it in class already, but I want to engrave it in my head with a reasoning so that I make sure to confirm I'm using promise in the future. 

Why should we use the promise version for the node:fs?